Prac_06/guitars.py

```diff
@@ -0,0 +1,22 @@
+from Prac_06.guitar import Guitar
+
+guitars = []
+
+"""
+guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+
+"""
+print("My guitars!")
+name = input("Name: ")
+
+while name != "":
+    year = int(input("Year: "))
+    cost = int(input("Cost: $"))
```

Comment:
`cost = int(input("Cost: $"))` should be `cost = float(input("Cost: $"))` because the user can enter a number with decimals for the guitar cost